DirectProgramming/C++SYCL_FPGA/Tutorials/Features/IPAuthoring_interfaces_overview/handshake-control/Readme.md

```diff
@@ -0,0 +1,206 @@
+# Handshake-control
+This example demonstrate a streaming pipelined kernel with conduit data interfaces and implementing a simple vector addition.
+
+This design will show how to specify streaming invocation interface through kernel properties and how to specify conduit interface using `annotated_ptr`.
+
+## Invocation Interface - Streaming pipelined interface
+In this design, the `SimpleVAdd` kernel is using streaming pipelined invocation interface. This is explicitly specified though kernel properties. 
+
+To use kernel properties, include either of the following header files:
+```cpp
+#include <sycl/ext/intel/fpga_extensions.hpp>
+```
+Or
+```cpp
+#include <sycl/ext/intel/experimental/fpga_kernel_properties.hpp>
+```
```

Comment:
```suggestion
```

if it's optional and fpga_extensions.hpp works, then don't make it complicated and just rely on fpga_extensions (which the user is already lincluding anyway)